packages/tools/bcm2835-bootloader/files/3rdparty/bootloader/config.txt

```diff
@@ -108,3 +108,10 @@
 # all values below this line were inserted from config.txt.bk (your old config)
 # and can be merged with the above values
 ################################################################################
+
+################################################################################
+# Include distribution specific config file if it exists.
+################################################################################
+
+[all]
+include distroconfig.txt
```

Comment:
Not entirely sure about this one. I think you should be adding this to your own DISTRO specific config.txt. However, if the `include` causes no error if it doesn't exist, then maybe there's no harm (and means a `distroconfig.txt` could be added later, if required). Worst case, we add a "common" distroconfig.txt that is blank?